### PR TITLE
feat(core): add README.md for connectors

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Add README.md placeholder for connectors.
(Did not consider multiple-language support at this stage.)

## Linear Issue Reference

LOG-1809

## Testing

UT.

### CODE REVIEW

File: packages/core/src/connectors/github/index.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import * as fs from 'fs';
```
Comment (IceHe) [inline packages/core/src/connectors/github/index.ts:1]: just need to import readFileSync?

Comment (darcyYe) [reply packages/core/src/connectors/github/index.ts]: updated.

File: packages/core/src/lib/passcode.test.ts [hunk 2]
```diff
@@ -134,6 +134,7 @@ describe('sendPasscode', () => {
         name: {},
         logo: '',
         description: {},
+        readme: '',
```
Comment (IceHe) [inline packages/core/src/lib/passcode.test.ts:137]: missing `'README.md'`?

File: packages/core/src/connectors/aliyun-sms/index.ts [hunk 3]
```diff
@@ -27,6 +29,7 @@ export const metadata: ConnectorMetadata = {
     'zh-CN':
       '短信服务（Short Message Service）是指通过调用短信发送API，将指定短信内容发送给指定手机用户。',
   },
+  readme: readFileSync('./README.md', 'utf8'),
```
Comment (wangsijie) [inline packages/core/src/connectors/aliyun-sms/index.ts:32]: we can check if this file is exists, prevent crashing.

**Review by wangsijie — CHANGES_REQUESTED** [review]

File: packages/core/src/lib/passcode.test.ts (hunk 2 above)
Comment (darcyYe) [reply packages/core/src/lib/passcode.test.ts:137]: This mocks the return value of some methods, anything follows the type define should be good enough.

File: packages/core/src/connectors/aliyun-sms/index.ts (hunk 3 above)
Comment (darcyYe) [reply packages/core/src/connectors/aliyun-sms/index.ts]: Updated, PTAL.

**Review by wangsijie — CHANGES_REQUESTED** [review]
Is safer to use `__dirname`:

```
existsSync(path.join(__dirname, 'README.md'));
```

**Review by wangsijie — APPROVED** [review]

**Pull request merged**